packages/fifteen-logic/Jakefile.js

```diff
@@ -0,0 +1,34 @@
+(function() {
+  'use strict'
+
+  require('shelljs/global')
+
+  desc('Run TypeScript Compiler')
+  task('build', function() {
+    exec('node_modules/.bin/tsc')
+    console.log('\n\ntsc complete')
+  })
+
+  desc('Run tests with AVA following tsc')
+  task('test', ['build'], function() {
+    exec('node_modules/.bin/ava dist/*.js')
```

Comment:
it looks like we'll need to use 'dist/*.test.js' and 'dist/**/*.test.js' b/c ava is trying to run tests for every js file right now